lib/msal-node/src/config/Configuration.ts

```diff
@@ -157,11 +161,15 @@ export function buildAppConfiguration({
     system,
     telemetry
 }: Configuration): NodeConfiguration {
+    const systemOptions: Required<NodeSystemOptions> = {
+        ...DEFAULT_SYSTEM_OPTIONS,
+        networkClient: system?.customAgentOptions ? new HttpClient(system.customAgentOptions as http.AgentOptions | https.AgentOptions) : new HttpClient(),
```

Comment:
If the app developer configures their own HttpClient, will that scenario still work?

Reply:
If I inject a custom HTTP client AND customAgentOptions, what will happen? Is `customAgentOptions` ignored ( I assume so).

Reply:
> If the app developer configures their own HttpClient, will that scenario still work?

What scenario?

Reply:
> If I inject a custom HTTP client AND customAgentOptions, what will happen? Is `customAgentOptions` ignored ( I assume so).

If the developer injects their own custom HttpClient, then the customAgentOptions are ignored/never used. In that case, the developer can implement their own HttpAgent.

Reply:
I can confirm that a developer's custom HttpClient is still used even when CustomAgentOptions are included in the system config.